<launch>

    <arg name="simulator_config" default="$(find ros_simulator)/config/pacejka_car_simulator.yaml" doc = "Path to simulator.yaml config. "/>

    <arg name="control_input" default="control_input"/>
    <arg name="vicon_topic" default="vicon"/>
    <arg name="imu_topic" default="imu"/>
    <arg name="gt_state_topic" default="gt_state"/>


    <node pkg="ros_simulator" name="ros_simulator" type="simulation_node" output="screen">
        <rosparam command="load" file="$(arg simulator_config)" />
        <remap from = "~/control_input" to = "$(arg control_input)" />
        <remap from = "~/vicon" to = "$(arg vicon_topic)" />
        <remap from = "~/imu" to = "$(arg imu_topic)" />
        <remap from = "~/gt_state" to = "$(arg gt_state_topic)" />
    </node>


</launch>